invenio_orcid/config.py

```diff
@@ -26,6 +26,8 @@
 
 ORCID_SYNCHRONIZATION_ENABLED = True
 
+ORCID_JSON_CONVERTER_MODULE = 'invenio_orcid.utils:convert_to_orcid'
```

Comment:
Can you have a look at Invenio-Records-REST serializers and use compatible API so one can reuse it in more places?
- ``ORCID_RECORD_SERIALIZER`` (e.g. https://github.com/inveniosoftware/invenio-records-rest/blob/master/invenio_records_rest/config.py#L48)
- API (see https://github.com/inveniosoftware/invenio-records-rest/blob/master/invenio_records_rest/serializers/json.py)